lib/beaker/options/parser.rb

```diff
@@ -165,6 +166,34 @@ def initialize
         @command_line_parser = Beaker::Options::CommandLineParser.new
         @presets             = Beaker::Options::Presets.new
         @validator           = Beaker::Options::Validator.new
+        @attribution         = Beaker::Options::OptionsHash.new
```

Comment:
@ferglor I thought you were going to encapsulate the attribution work within the `OptionsHash` object? Did you mean that you'd just use an `OptionsHash` object to keep track of attribution then?